core/server/api_container/server/startosis_engine/kurtosis_instruction/add_service/add_services.go

```diff
@@ -123,25 +124,48 @@ func (builtin *AddServicesCapabilities) Execute(ctx context.Context, _ *builtin_
 		renderedServiceConfigs[renderedServiceName] = renderedServiceConfig
 	}
 
-	startedServices, failedServices, err := builtin.serviceNetwork.AddServices(ctx, renderedServiceConfigs, parallelism)
+	serviceToUpdate := map[service.ServiceName]*service.ServiceConfig{}
+	serviceToCreate := map[service.ServiceName]*service.ServiceConfig{}
```

Comment:
Oooh good catch, and I think it's linked to your remark below. I basically need to pass it to the call to `builtin.serviceNetwork.AddServices` a few lines below, instead of `renderedServiceConfigs` 🤦 🤦